src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -180,16 +184,30 @@ def get(self, request: Request, organization) -> Response:
             return Response({})
 
         use_rpc = request.GET.get("useRpc", "0") == "1"
+        transform_alias_to_input_format = (
+            request.GET.get("transformAliasToInputFormat") == "1" or use_rpc
+        )
```

Comment:
Are we not able to just force this on always instead of needing the param

Reply:
You tell me, honestly! I followed what the other endpoints do. For EAP, it's always on, for indexed spans, there's a parameter. I _think_ that'll make it easier to coordinate switching over, but it's probably a small difference

Reply:
I'd prefer if we can move away from needing this parameter at all, if the frontend doesn't need it un-transformed lets just not use this? ie. does the frontend need the old format still?

Reply:
🤔 well, the meta isn't in use at all right now, so I guess I could just force it, yeah! I'll probably replacing all the frontend usages of this at the same time anyway. I'll get rid of it 👍🏻